### DISCUSSION TITLE
[Feature Request] 为 code 标签添加 skip、only 标识

### DISCUSSION THREAD

**Discussion by Wxh16144** [umijs/dumi#1600] [Ideas]
### 概述

为 `<code src=“./your/patch/demo.tsx”/>` 标签添加类似 jest 测试时的 only、skip 标识

### 动机

一个场景是在维护 antd 的 `form/index.zh-CN.md` 时，里面有很多 code 标签引入外部 demo。有时候可能某些 demo 报错，亦或者 demo 太多了需要进行控制。

以往的操作是在 md 文件中注释掉，

### 实现

和  jest 单元测试的 case 类似：
-  **only**：表示仅运行当前文件中这个 code 标签
- **skip**：表示跳过当前文件中这个 code 标签

可以加入一个逻辑：`skip` 和 `only` 仅在非生产环境中生效，构建生产环境的时候如果存在标识则在日志中输出警告信息。

细节：TBD

### 使用

```html
<code  src=“./demo1.tsx” skip ></code>  <!— 跳过这个 demo 解析 —>
<code  src=“./demo2.tsx” ></code>
<code  src=“./demo3.tsx” ></code>
<code  src=“./demo4.tsx” ></code>
<code  src=“./demo5.tsx” ></code>
```

### 缺点

无

**Comment by PeachScript** [comment]
很棒的想法，可以搞起来👍

另外可以思考下代码块 demo 要不要支持对应的 frontmatter 或者 modifier，类似：
<pre lang="markdown">
```jsx | only
...
```

or

```jsx
/**
 * only: true
 */
...
```
</pre>

**Reply by Wxh16144** [reply]
我比较偏向于第一种方案:

<pre lang="markdown">
```jsx | only
// ...
```
</pre>

它不像 `title` 、`compact` 等配置会作用于生产环境，即使开发者需要生产环境跳过某些 code 也可以使用 `debug` 过滤。 skip、only 仅方便开发的时候使用。

**Reply by PeachScript** [reply]
可以的，先只支持 code 也问题不大，代码块实现起来会麻烦一些
